client/component/App.jsx

```diff
@@ -23,7 +24,13 @@ class App extends React.Component {
 
     render() {
         return (
-            <>React App is working. Please continue your work.</>
+            <>
+            {Object.keys(this.state.product).length > 0 &&
+                <Price product={this.state.product}/> }
+            {Object.keys(this.state.product).length === 0 &&
```

Comment:
would like to ask for an additional explanation here.

Reply:
Because my this.state.product is object, there is not an explicit way for me to check if the object is empty. So instead, I grab the keys as an array (with Object.keys method) and check if the keys array is empty. If it is, I know the product object is empty, so I render the div belong. If it is filled (from componentDidMount working properly), then I know I can display the Price component